src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
```

Comment:
Don't use verbs for class/interface names. Consider renaming `FigureSupplier` to something that represents its purpose without using verbs, like `FigureFactory` or `RandomFigureGenerator`.